src/style.css

```diff
@@ -0,0 +1,144 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  background: #e5e5e5;
+}
+
+.card {
+  width: 200px;

```

Comment:
You can't set width to the BEM block
Create additional class to make this block the element of body

Reply:
You can't set width to the BEM block